### ISSUE TITLE
Questions regarding Mattel Games

### ISSUE THREAD

**Issue by DonaldTsang** [datamllab/rlcard#65]
Why is Uno in the list when Skip-Bo https://en.wikipedia.org/wiki/Skip-Bo and Phase 10 https://en.wikipedia.org/wiki/Phase_10 isn't, when in fact they are popular as well?

**Comment by TomLucidor** [comment]
Right, but what are the subtle differences? Can it have a little more description?
